### ISSUE TITLE
openvidu-browser 2.21 error for external coturn when not publishing

### ISSUE THREAD

**Issue by basicmonkey** [OpenVidu/openvidu#714]
**Describe the bug**
Using openvidu-browser v2.21.1 with OpenVidu 2.21 and external coturn as installed using OV 2.21 instructions. Works great.

For users who are publishing, there are no errors in the log. For OV objects that aren't publishing, subscribe only, an error is generated in the logs.

**Expected behavior**
No error generated for non publisher objects. Maybe check for publish state?

**Wrong current behavior**
Error appears in logs:

```
openvidu-browser-2.21.0.min.js:1 ERROR:Request to onIceCandidate for connection con_DeqLFTxkLx gone wrong. There is no publisher endpoint available. Code: 408 in Request: method:onIceCandidate params:{"endpointName":"xxx","candidate":"candidate:1066253390 1 udp 8331007 xxx 52206 typ relay raddr xxx rport 55620 generation 0 ufrag 9W1s network-id 1","sdpMid":"0","sdpMLineIndex":0} request:undefined
OpenViduLogger.error @ openvidu-browser-2.21.0.min.js:1
openvidu-browser-2.21.0.min.js:1 Error sending ICE candidate: {"code":408,"message":"Request to onIceCandidate for connection con_DeqLFTxkLx gone wrong. There is no publisher endpoint available. Code: 408","sessionId":"xxx","requestTime":1649789507144}
```

**OpenVidu tutorial where to replicate the error**

**OpenVidu deployment info**
v2.21 deployed on Ubuntu 20.04 on vSphere stack
External coturn deployed using latest 2.21 instructions

**Client device info (if applicable)**
Chrome 96-100 on Mac and Windows

**Screenshots**

**Additional context**
```
openvidu-browser-2.21.0.min.js:1 ERROR:Request to onIceCandidate for connection con_DeqLFTxkLx gone wrong. There is no publisher endpoint available. Code: 408 in Request: method:onIceCandidate params:{"endpointName":"xxx","candidate":"candidate:1066253390 1 udp 8331007 xxx 52206 typ relay raddr xxx rport 55620 generation 0 ufrag 9W1s network-id 1","sdpMid":"0","sdpMLineIndex":0} request:undefined
OpenViduLogger.error @ openvidu-browser-2.21.0.min.js:1
openvidu-browser-2.21.0.min.js:1 Error sending ICE candidate: {"code":408,"message":"Request to onIceCandidate for connection con_DeqLFTxkLx gone wrong. There is no publisher endpoint available. Code: 408","sessionId":"xxx","requestTime":1649789507144}
```

**Comment by cruizba** [comment]
Hello @basicmonkey , can you replicate the same error in a tutorial or give us an app example?

I've never seen this error before

**Comment by pabloFuente** [comment]
After some quick research, we have come to the conclusion that this error has already been addressed in the master branch, and will be included in release 2.22.0 (coming soon). The commit with the fix is the following one: https://github.com/OpenVidu/openvidu/commit/87f2613860995e9e13bb8379e87ac7f0b48d387c

This is not a fatal error. It does not really affect the behavior of the application in any way other than the error being logged.

**Comment by basicmonkey** [comment]
Great stuff. Did look like it just needed a check to see if the OV object is a publisher. Thank you both.
